site/docs/components/overlay/examples.mdx

```diff
@@ -0,0 +1,61 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="overlay" exampleName="Default">
+
+### Default
+
+The basic overlay component allows the user to expose and interact with supplemental information. The overlay remains persistent until the user dismisses it.  
+
+You can configure the `placement` of an overlay to the top, bottom, left or right in relation to a UI element.  
+
+</LivePreview>
+
+<LivePreview componentName="overlay" exampleName="OverlayRight" >
```

Comment:
Do we really need an example per placement? This doesn't align with our Tooltip docs page

Reply:
I'll work on an example similar to what we have on Tooltip's docs!